CONTRIBUTING.md

```diff
@@ -9,3 +9,26 @@ https://qiskit.org/documentation/contributing_to_qiskit.html
 
 In addition to the general guidelines there are specific details for
 contributing to qiskit_sphinx_theme, these are documented below.
+
+------
```

Comment:
Are these vertical separators deliberate?  I'm not complaining if so, just wondering if they're meant to be there.

Reply:
oh yeah they're intentional! I love a line separator 😄